src/csvcubed/utils/skos/codelist.py

```diff
@@ -63,6 +65,32 @@ def get_codelist_col_title_by_property_url(
     return results[0].column_title
 
 
+def get_codelist_notation_col_title_from_primary_key(
```

Comment:
Again, we can't call this the `notation` column anymore. It's really the unique identifier column.